lib/cli/commands/config.js

```diff
@@ -0,0 +1,76 @@
+import path from "node:path";
+import os from "node:os";
+import chalk from "chalk";
+import baseMiddleware from "../middlewares/base.js";
+import Configuration from "@ui5/project/config/Configuration";
+
+const ALLOWED_KEYS = ["mavenSnapshotEndpointUrl"];
+const ui5RcFilePath = path.resolve(path.join(os.homedir(), ".ui5rc"));
```

Comment:
Can I set the config path to be project specific?

Reply:
Not as of today

Reply:
Ack